src/components/EditableField/EditableField.module.scss

```diff
@@ -0,0 +1,42 @@
+.editableFieldContainer {
+  width: 100%;
+}
+
+.editableField {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  width: 100%;
+  gap: 0.5rem;
+}
+
+.editInput {
+  background: none;
+  border: 2px solid $light-grey;
+  border-radius: 4px;
+  width: 100%;
+  font-size: 1.8rem;
+  font-weight: $extrabold;
+}
+
+.editTextarea {
+  background: none;
+  border: 2px solid $light-grey;
+  border-radius: 4px;
+  width: 100%;
```

Comment:
Having 4 and 4 lines duplicated is fine. Just remember, there are ways to abstract CSS. Either additional classname that you will pass to both inputs. Or scss has @extend

![CleanShot 2025-01-15 at 10 31 16](https://github.com/user-attachments/assets/4f73aec0-2212-4ca6-99f2-0bde0cca0893)